model/integration_action.go

```diff
@@ -25,15 +25,49 @@ const (
 
 type DoPostActionRequest struct {
 	SelectedOption string `json:"selected_option"`
+	Cookie         string `json:"cookie,omitempty"`
 }
 
 type PostAction struct {
-	Id          string                 `json:"id"`
-	Name        string                 `json:"name"`
-	Type        string                 `json:"type"`
-	DataSource  string                 `json:"data_source"`
-	Options     []*PostActionOptions   `json:"options"`
+	// A unique Action ID. If not set, generated automatically.
+	Id string `json:"id"`
+
+	// The type of the interactive element. Currently supported are
+	// "select" and "button".
+	Type string `json:"type"`
+
+	// The text on the button, or in the select placeholder.
+	Name string `json:"name"`
```

Comment:
Maybe use `omitempty`? We discussed this briefly in the dev meeting and didn't found any downsides.